.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+- [ ] Closes #xxx
```

Comment:
One drawback to this bullet is that it will automatically link the issue to be closed when the PR is merged.  Right now we don't close issues, we mark them as "fixed in development".  See https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

Reply:
Yeah let's take that out. Keywords are cool, but I prefer manual closing because it forces a human to ask themselves "has this issue been handled". Also doesn't work great with our development branch workflow as you point out.